views/result.erb

```diff
@@ -109,7 +109,7 @@
                   <div class="hit-links text-right">
                     <label>
                       <input
-                      type="checkbox" value=<%="#{ hit.accession }"%>>
+                      type="checkbox" value="<%="Query_#{query.number}_hit_#{hit.number}"%>">
```

Comment:
Why not set the value directly to '#Query_1_hit_1'? Like BS does it for `data-target`?